inc/compat/themes/compat-theme-buddyboss-theme.php

```diff
@@ -0,0 +1,40 @@
+<?php
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Compatibility with theme: Avada (by ThemeFusion).
+ */
+class FluidCheckout_ThemeCompat_Buddyboss extends FluidCheckout {
```

Comment:
Minor: Class name should follow CamelCase on the component part. `FluidCheckout_ThemeCompat_BuddyBoss` instead of `FluidCheckout_ThemeCompat_Buddyboss`.